apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -182,8 +182,14 @@ func (in *K8ssandraCluster) GetInitializedDatacenters() []CassandraDatacenterTem
 // SanitizedName returns a sanitized version of the Cassandra cluster name override if it exists,
 // otherwise the k8c object name.
 func (in *K8ssandraCluster) SanitizedName() string {
+	return cassdcapi.CleanupForKubernetes(in.CassClusterName())
+}
+
+// CassClusterName returns a sanitized version of the Cassandra cluster name override if it exists,
```

Comment:
I guess the comment is not quite accurate here ;)

Reply:
No it's not 😬